model_api/cpp/models/include/models/image_model.h

```diff
@@ -48,10 +48,11 @@ class ImageModel : public ModelBase {
     static std::vector<std::string> loadLabels(const std::string& labelFilename);
     std::shared_ptr<ov::Model> embedProcessing(std::shared_ptr<ov::Model>& model,
                                                     const std::string& inputName,
-                                                    const ov::Layout& = ov::Layout("NCHW"),
-                                                    RESIZE_MODE resize_mode = NO_RESIZE,
-                                                    const cv::InterpolationFlags interpolationMode = cv::INTER_LINEAR,
-                                                    const ov::Shape& targetShape = ov::Shape(),
+                                                    const ov::Layout&,
+                                                    RESIZE_MODE resize_mode,
+                                                    const cv::InterpolationFlags interpolationMode,
+                                                    const ov::Shape& targetShape,
+                                                    uint8_t pad_value,
```

Comment:
I am curious if `pad_value` is always `uint8_t`.

Reply:
It is treated as u8: `opset10::Constant::create(element::u8, Shape{}, {pad_value})`. So I endorse this for wrappers